README.md

```diff
@@ -102,7 +102,7 @@ Having a bug tracking system doesn't make bug handling efficient and CI/CD doesn
 - Walk me through a typical sprint on this team
 - How do you balance technical vs business goals?
 - How do you share knowledge?
-- How big are the teams?
+- How are teams structured and how many services is one in charge of?
```

Comment:
Thanks. This feels like a separate but also interesting question. Would you like to add it below the one about team sizes?

Reply:
Sounds great! comitting